website_base_multi_image/controllers/main.py

```diff
@@ -0,0 +1,57 @@
+# -*- coding: utf-8 -*-
+from openerp.http import request, STATIC_CACHE
+from openerp.addons.web import http
+from openerp import _
+import logging
+logger = logging.getLogger(__name__)
+
+
+class Web_Editor(http.Controller):
+
+    #------------------------------------------------------
+    # remove images
+    #------------------------------------------------------
+    @http.route('/website/image/remove', type='json', auth='user')
```

Comment:
This one controller could be replaced by [a JS call to `unlink`](https://github.com/odoo/odoo/blob/8e65b57df796860a008270d8b425711d2bf88888/addons/web/static/src/js/framework/data.js#L488-L495) too. Not so hard to do, and I believe it's good to get used to use built-in standard RPC methods instead of creating a controller for everything.